articles/azure-monitor/app/opencensus-python.md

```diff
@@ -22,28 +23,21 @@ You may have noted that OpenCensus is converging into [OpenTelemetry](https://op
 
 ## Prerequisites
 
-- An Azure subscription. If you don't have an Azure subscription, create a [free account](https://azure.microsoft.com/free/) before you begin.
-- Python installation. This article uses [Python 3.7.0](https://www.python.org/downloads/release/python-370/), although other versions will likely work with minor changes. The Opencensus Python SDK only supports Python v2.7 and v3.4+.
-- Create an Application Insights [resource](./create-new-resource.md). You'll be assigned your own instrumentation key (ikey) for your resource.
-
-[!INCLUDE [azure-monitor-log-analytics-rebrand](../../../includes/azure-monitor-instrumentation-key-deprecation.md)]
+You can find all of the necessary resources to run these examples in the sample Azure Monitor OpenCenses Python git repository located [here](https://github.com/Azure-Samples/azure-monitor-opencensus-python/tree/master/docs_samples).
```

Comment:
The following document and examples do not need all of the resources/prerequisites that are defined in the sample apps so I don't believe we want this change. Also, I don't think we want the message of ikey deprecation to be removed.

I think you can simply add a message similarly to the other files like:

`You can find more examples of applications using this SDK in the sample Azure Monitor OpenCensus Python samples repository located [here](https://github.com/Azure-Samples/azure-monitor-opencensus-python/blob/master/docs_samples/01_python_app_insights_overview.md).